docs/content/config.md

```diff
@@ -77,6 +77,13 @@ resolver:
 # Optional: set to skip running `go mod tidy` when generating server code
 # skip_mod_tidy: true
 
+# Optional: set to modify the initialisms regarded for Go names
+#go_initialisms:
+#  replace_defaults: true # if true, the default initialisms will get dropped in favor of the new ones instead of being added
```

Comment:
Again, I would make this value start as false for copy-paste, but good comment.

Reply:
Done!